modin/utils.py

```diff
@@ -375,7 +375,7 @@ def wrapper(*args, **kwargs):
 
 def get_current_backend():
```

Comment:
```suggestion
def get_current_factory():
```
let's be consistent

Reply:
Everywhere outside the `factories` module we use the term "backend", It may be confusing to met "factory" somewhere in the code where there is no backend factories or factories dispatching and so no need to use this term. I would suggest changing the doc of this func to the following instead:
"Return current backend name as a string."

Reply:
This is the same ambiguity as in my first comment... I don't like it but I won't insist on changing it here.